app/web/src/features/dashboard/constants.ts

```diff
@@ -4,14 +4,13 @@ export const FILL_IN_WHO_I_AM =
   '1. Fill in your "Who I am" section with a few sentences about yourself';
 export const UPLOAD_PHOTO =
   "2. Upload a photo (by clicking on the avatar on the edit page)";
-export const CLICK_HERE_TO_EDIT = "Click here to edit your profile";
+export const CLICK_HERE_TO_EDIT = "Edit your profile";
 export const DONT_YOU_HATE =
   "Don't you hate how other platforms have empty \"ghost\" profiles with no information. Please don't ghost us! 👻👻👻";
 
-export const PASSWORD_TEXT_1 = "You can set a password on";
-export const PASSWORD_TEXT_LINK = "the account settings page";
-export const PASSWORD_TEXT_2 =
-  "so you don't need check your email each time you log in.";
+export const PASSWORD_TEXT_1 =
+  "Tip: You can set a password so you don't need check your email each time you log in";
```

Comment:
```suggestion
  "You can set a password so you don't need check your email each time you log in";
```
Just because it's not really a tip at the moment, this never goes away if you don't set a password :P